windows/CodePush/UpdateManager.cs

```diff
@@ -98,6 +98,15 @@ internal class UpdateManager
                 await downloadFile.RenameAsync(expectedBundleFileName).AsTask().ConfigureAwait(false);
                 await downloadFile.MoveAsync(newUpdateFolder).AsTask().ConfigureAwait(false);
             }
+            catch (Exception e)
+            {
+				/*TODO: ZipFile.ExtractToDirectory is not reliable and throws exceptions if:
+				 	- folder exists already
+					- path is too long
+					it needs to be handled			 
+				 */
+                throw e;
+            }
```

Comment:
This will screw up the stack trace of the exception (the point where the exception is thrown is recalculated from here).   Since this has no net affect on the behavior other, please remove this catch block.